src/components/core/Error/Error.tsx

```diff
@@ -0,0 +1,11 @@
+import classes from './Error.module.scss';
+
+interface ErrorProps {
+  error?: string;
+}
+
+const Error: React.FC<ErrorProps> = ({ error }) => {
+  return error && <span className={`${classes['c-error']}`}>{error}</span>;
+};
+
+export default Error;
```

Comment:
I would suggest naming this component as ValidationMessage and making it more general